atlp-rwanda/cabal-phantom-backend
# #ZM0sHy6Z Set up sequelize

### **What about this PR?**
This PR is about to configure the app to use Postgres database

### **Description of Task to be completed**
- Set up the migrations and models structure
- Dummy user table to test configuration

### **How should be manually tested?**
- Clone the repo
- checkout the branch `ch-set-up-sequelize-ZM0sHy6Z`
- Install all packages
- Create two Postgres databases one for test and the other for development and assign the values of the connection strings to DEV_DATABASE_URL and TEST_DATABASE_URL
- create **`.env`** file and insert those database's connection string
- Run this command `npm run migrate` to make migration or Run `npm run migrate:reset` to undo migration 
- Run this command `npm run seeding` to seed the database with dummy data
- You may check the data in your Postgres Db (Optionally)

### **Relevant pivots tracker**
[#ZM0sHy6Z](https://trello.com/c/ZM0sHy6Z/17-configure-the-app-to-use-postgres-database)
